contracts/cw20-ics20/src/ibc.rs

```diff
@@ -32,15 +32,20 @@ pub struct Ics20Packet {
     pub receiver: String,
     /// the sender address
     pub sender: String,
+    /// used only by us to control ack handling
+    pub v: Option<u32>,
```

Comment:
I created an issue about this #662

Reply:
Yeah, I found this and had to revert it.
Turns out ibctransfer packet uses a super-strict JSON parser that fails on any unknown fields.
It seems to me that this is undesired behavior, but that is another discussion.

I need a different format and migration step to do a new release of this, very correct. I was out for a while.